addons/sms/static/tests/web/sms_button.test.js

```diff
@@ -0,0 +1,130 @@
+import {
+    click,
+    contains,
+    editInput,
+    start,
+    startServer,
+} from "@mail/../tests/mail_test_helpers";
+import { describe, expect, test } from "@odoo/hoot";
+import { queryFirst } from "@odoo/hoot-dom";
+import { defineSMSModels } from "@sms/../tests/sms_test_helpers";
+import { mockService, mountView } from "@web/../tests/web_test_helpers";
+
+describe.current.tags("desktop");
+defineSMSModels();
+
+test("Sms button in form view", async () => {
+    await start();
```

Comment:
Is this `start` really required? It looks like we only mount a view which is not linked to the mail module in any way? Same for the ones below?